dev/com.ibm.ws.jdbc_fat_krb5/fat/src/com/ibm/ws/jdbc/fat/krb5/containers/KerberosContainer.java

```diff
@@ -52,11 +49,7 @@
 
     // NOTE: If this is ever updated, don't forget to push to docker hub, but DO NOT overwrite existing versions
     private static final String IMAGE = "aguibert/krb5-server:1.2";
```

Comment:
Similar to previous comment.  Consider externalizing this value (via public), if used elsewhere, so it can be used instead of a hardcoded value in other classes.